addons/account/tests/test_early_payment_discount.py

```diff
@@ -226,6 +226,34 @@ def test_register_discounted_payment_on_single_out_invoice_with_tax(self):
             {'amount_currency': 1552.5, 'tax_tag_invert': False},
         ])
 
+    def test_register_discounted_foreign_currency_payment_on_invoice_with_tax(self):
+        self.env.company.early_pay_discount_computation = 'included'
+        inv_1500_10_percents_discount_tax_incl_15_percents_tax = self.env['account.move'].create({
```

Comment:
```suggestion
        invoice = self.env['account.move'].create({
```
It's quite explicit it's an invoice-with-1500-but-a-tax-and-also-an-early-payment-but-price-included-blablabla